src/ast/ReturnDecl.ts

```diff
@@ -23,7 +23,7 @@ export default class ReturnDecl extends AstNode {
     }
 
     public typeCheck(): void {
-        const wasDeclared: boolean = this.typeTable.isValidType(this.returnType);
+        const wasDeclared: boolean = this.typeTable.isValidType(this.returnType.replace(/[\[\]]/g, ""));
```

Comment:
I'm assuming this line of code just "unwraps" the collection? i.e. get `number` from `number[]` by killing the brackets?

Reply:
yep- that's right!